proposals/source_tiktok_proposal.md

```diff
@@ -0,0 +1,77 @@
+# Source Tiktok Proposal
+
+## Description
+
+The Tiktok Source connector is used to Receive events from TikTok when a user’s post is liked, shared, or commented on, and when a user is followed and put the data into a CloudEvent to send it to the target
+This page describes the design of the Tiktok Source in detail.
+
+## Programming Language
+
+-[ ] Golang
+-[x] Java
+
+## Prerequisites
+
+- A Tiktok WebHook suscription
+- SSL signed certificate
+
+## Connector Details
+
+### Configuration
+
+The Tiktok event have the following specification:
+
+| Key               | Type     | Description                                                                  |
+| :---------------- | :------- | :----------------------------------------------------------------------------|
+| client_key        | string   | The unique identification key provisioned to the partner.                    |
+| event             | string   | Event name.                                                                  |
+| create_time       | int64    | The time in which the event occurred. UTC epoch time is in seconds.          |
```

Comment:
this is the config to start the connector

Reply:
why would you need the user to input a create time

specification should be made of
 
key , passwords username etc ...